src/GraphQL/Language/AST/Argument.cs

```diff
@@ -16,6 +16,10 @@ public Argument(NameNode name)
 
         public string Name => NamedNode?.Name;
         public NameNode NamedNode { get; }
+
+        public string Comment => CommentNode?.Comment;
+        public CommentNode CommentNode { get; set; }
```

Comment:
http://spec.graphql.org/June2018/#sec-Source-Text.Ignored-Tokens

> Before and after **every** lexical token may be any amount of ignored tokens including WhiteSpace and Comment.

So I propose to move these properties
```c#
        public string Comment => CommentNode?.Comment;
        public CommentNode CommentNode { get; set; }
```

in `AbstractNode`.  So there will be less public api changes.

Reply:
CommentNode inherits from AbstractNode, so we cannot move this in AbstractNode

Reply:
Why not?

Reply:
Done